proto/cusf/mainchain/v1/wallet.proto

```diff
@@ -56,6 +65,30 @@ message CreateNewAddressResponse {
   string address = 1;
 }
 
+message CreateSidechainProposalRequest {
+  google.protobuf.UInt32Value sidechain_id = 1;
+  SidechainDeclaration declaration = 2;
+}
+
+message CreateSidechainProposalResponse {
```

Comment:
Same Q goes here for wrappers

Reply:
https://github.com/LayerTwo-Labs/cusf_sidechain_proto/pull/14#discussion_r1820474787